contributor-license-agreements/LogSearch_Entity_Contributor_License_Agreement.md

```diff
@@ -1,216 +1,192 @@
-(including any right to adopt any future version of a license if permitted):
-Creative Commons Attribution - https://creativecommons.org/licenses/by/4.0
-
-2.4 Moral Rights. If moral rights apply to the Contribution, to the maximum
-extent permitted by law, You waive and agree not to assert such moral rights
-against Us or our successors in interest, or any of our licensees, either
-direct or indirect.
-
-2.5 Our Rights. You acknowledge that We are not obligated to use Your
-Contribution as part of the Material and may decide to include any
-Contribution We consider appropriate.
-
-2.6 Reservation of Rights. Any rights not expressly licensed under this
-section are expressly reserved by You.
-
-## 3. Agreement
-
-You confirm that:
-
-(a) You have the legal authority to enter into this Agreement.
-
-(b) You own the Intellectual Property Rights assigned and or licensed under 
-this Agreement;
-
-(c) The grant of rights under Section 2 does not violate any grant of rights
-which You have made to third partiest; 
-
-(d) You have followed the instructions in https://github.com/logsearch/licensing,
-if You do not own the Intellectual Property Rights in the entire work of 
-authorship Submitted.
-
-## 4. Disclaimer
-
-EXCEPT FOR THE EXPRESS WARRANTIES IN SECTION 3, THE CONTRIBUTION IS PROVIDED
-"AS IS". MORE PARTICULARLY, ALL EXPRESS OR IMPLIED WARRANTIES INCLUDING,
-WITHOUT LIMITATION, ANY IMPLIED WARRANTY OF MERCHANTABILITY, FITNESS FOR A
-PARTICULAR PURPOSE AND NON-INFRINGEMENT ARE EXPRESSLY DISCLAIMED BY YOU TO US.
-TO THE EXTENT THAT ANY SUCH WARRANTIES CANNOT BE DISCLAIMED, SUCH WARRANTY IS
-LIMITED IN DURATION TO THE MINIMUM PERIOD PERMITTED BY LAW.
-
-4.1. Limitations and Exclusions
-Nothing in this Agreement shall exclude or limit the liability of You or Us 
-for acts of fraud (including fraudulent misrepresentation) or for death 
-or personal injury caused by negligence.
+```
+   Thank you for your interest in The Logsearch Project (the "Project"); 
+   run by GAIN Capital UK Ltd; whoes source code is published at http://github.com/logsearch.
+   
+   In order to clarify the intellectual property license
+   granted with Contributions from any person or entity, the Project
+   must have a Contributor License Agreement (CLA) on file that has been
+   signed by each Contributor, indicating agreement to the license terms
+   below. This license is for your protection as a Contributor as well
+   as the protection of the Project and its users; it does not change
+   your rights to use your own Contributions for any other purpose.
+
+   This version of the Agreement allows an entity (the "Corporation") to
+   submit Contributions to the Project, to authorize Contributions 
+   submitted by its designated employees to the Project, and to grant 
+   copyright and patent licenses thereto.
+
+   If you have not already done so, please complete and sign, then scan and
+   email a pdf file of this Agreement to david.laing@labs.cityinde.com
+   
```

Comment:
@mrdavidlaing this email address looks wrong. Should we create a new email address: opensource@gaincapital.com ?